### PR TITLE
fix: shm size for triton server

### PR DESCRIPTION
**Pull request by Phelan164**
Because

- Triton server need at least 1GB shm size for loading some Python backend

This commit

- Update docker-compose for shm size

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -169,3 +169,72 @@ services:
       - ALLOW_EMPTY_PASSWORD=yes
     ports:
       - 6379:6379
+
+  database:
```
Comment (bochengyang) [inline docker-compose.yml:173]: This service has another `pg_sql` run in this docker-compose, please remove it

File: docker-compose.yml [hunk 2]
```diff
@@ -169,3 +169,72 @@ services:
...
+      container_name: pg_sql
+      ports:
+          - 5432:5432
+      environment:
+          POSTGRES_PASSWORD: password
+      healthcheck:
+          test: ["CMD-SHELL", "pg_isready -U postgres"]
+          timeout: 20s
+          retries: 10
+  model-migrate:
```
Comment (bochengyang) [inline docker-compose.yml:184]: This is a duplicated service I think?

File: docker-compose.yml [hunk 3]
```diff
@@ -169,3 +169,72 @@ services:
...
+      environment:
+          CFG_DATABASE_HOST: pg_sql
+          CFG_HTTP_PORT: 8080
+          CFG_DATABASE_USERNAME: "${PG_SQL_USERNAME}"
+          CFG_DATABASE_PASSWORD: "${PG_SQL_PASSWORD}"
+          CFG_DATABASE_PORT: "${PG_SQL_PORT}"
+          CFG_DATABASE_VERSION: 1
+      entrypoint: ./migrate
+
+  model-backend:
```
Comment (bochengyang) [inline docker-compose.yml:200]: This is a duplicated service I think?

File: docker-compose.yml [hunk 4]
```diff
@@ -169,3 +169,72 @@ services:
...
+          CFG_DATABASE_VERSION: 1
+          CFG_TRITONSERVER_GRPCURI: triton-server:8001
+          CFG_TRITONSERVER_MODELSTORE: /models
+      restart: on-failure
+      ports:
+          - 8080:8080
+      volumes:
+          - models:/models
+
+  triton-server:
```
Comment (bochengyang) [inline docker-compose.yml:221]: This is a duplicated service I think?

**Comment by bochengyang** [comment]
Please close it and re-create or push it again after rebasing the latest main

File: docker-compose.yml (hunk 1 above)
Comment (Phelan164) [reply docker-compose.yml]: @bochengyang yes I used old version not rebase yet. I fixed

File: docker-compose.yml (hunk 4 above)
Comment (Phelan164) [reply docker-compose.yml]: save above

File: docker-compose.yml (hunk 2 above)
Comment (Phelan164) [reply docker-compose.yml]: save above

**Comment by Phelan164** [comment]
Make other PR

**Pull request closed without merging**